multicluster/cmd/multicluster-controller/options.go

```diff
@@ -40,6 +40,9 @@ type Options struct {
 	// The type of IP address (ClusterIP or PodIP) to be used as the Multi-cluster
 	// Services' Endpoints.
 	EndpointIPType string
+	// EnableStretchedNetworkPolicy will exchange labelIdentity info among the whole
+	// clusterSet.
```

Comment:
Enable StretchedNetworkPolicy to exchange labelIdentity info among the whole clusterSet.

Reply:
Done.